### ISSUE TITLE
empty else case not handled.

### ISSUE THREAD

**Issue by jomaway** [hikerpig/pintora#284]
if i end in and if case with an empty else case i have a break in the diagram. See screenshot

Would expect to get a line from the `gültig` to the small diamond.
```
partition Noten_einlesen {
  start
      :read input string from user;
      :split string on ',';
      while (iter liste) is (next)
        :note in int umwandeln;
        :note überprüfen;
        if (gültig) then
            :print error;
            end
        else (no)
        endif
      endwhile (done)
      :return liste;

  end

}
```

![Bildschirmfoto vom 2024-05-07 17-33-12](https://github.com/hikerpig/pintora/assets/22931557/600d4be6-0805-4858-8c6f-846bb574d27f)

**Comment by hikerpig** [comment]
I'm not sure if it's a web app cache thing, you may try `cmd+shift+R` to force refresh the page.
I do remember that I fixed this some months ago, raised in a similar issue. 

Here's my screenshot on the live-editor , the else line does show.
![image](https://github.com/hikerpig/pintora/assets/2259688/13146209-2df2-423f-b801-23c3cc32e715)

But hey! When I paste all your example code into the live-editor, an even more bizare layout bug strikes me , I need to fix this, thank you for the example.
![image](https://github.com/hikerpig/pintora/assets/2259688/d7638034-4dbb-449c-a0ca-cb1265d7228e)

**Comment by jomaway** [comment]
I used the online live editor. And now i get the same weird connections as in your screenshot

**Comment by hikerpig** [comment]
The strange line corner is fixed in @pintora/diagrams@0.7.4 , the website live editor is also updated.
